set.mm

```diff
@@ -35696,8 +35819,22 @@ practical reasons (to avoid having to prove sethood of ` A ` in every use
     sbc2ie.3 $e |- ( ( x = A /\ y = B ) -> ( ph <-> ps ) ) $.
     $( Conversion of implicit substitution to explicit class substitution.
        (Contributed by NM, 16-Dec-2008.)  (Revised by Mario Carneiro,
-       19-Dec-2013.) $)
+       19-Dec-2013.)  Reduce axiom usage and shorten proof.  (Revised by Gino
+       Giotto, 12-Oct-2024.) $)
```

Comment:
"Proof shortened by ..." instead of "Revised by ...": Only the proof has changed, not the theorem itself.

Reply:
> Only the proof has changed, not the theorem itself.

We use the "Revised by" tag when we reduce axiom usage, and reducing axiom usage involves the proof only, not the statement (most of the times).

I believe what you meant to say is that when a proof is shortened it doesn't matter whether it also reduces axiom usage or not, it counts as a shortening in all cases. If that's what you meant then I don't have a particular opinion about it, both are fine for me.

Reply:
I would prefer "Proof shortened by ..." if the proof is shorter (and the theorem is not changed), regardless axiom usage reduction. If the proof becomes longer after a reduction of axion usage, then "Revised by ..." is fine.

Reply:
Done. I also removed the revision comment since it would be redundant `...and shorten proof. (Proof shortened by..`.

Reply:
I won't insist on changing here, but to me, "revised by" denotes something more substantial than "proof shortened by", and axiom usage changes are in general more substantial than proof shortenings, and make brief explanatory comments more natural. The present case is typically a revision. The initial writing by @GinoGiotto was clearer.